seed/views/properties.py

```diff
@@ -762,8 +762,10 @@ def update(self, request, pk=None):
                         )
                         # Removing organization key AND import_file key because they're not JSON-serializable
                         # TODO find better solution
-                        result['state'].pop('organization')
-                        result['state'].pop('import_file')
+                        if 'organization' in result['state']:
```

Comment:
i think we can remove these lines all together now, but didn't want to yet. The serializer should handle it fine.